mmv1/third_party/terraform/transport/error_retry_predicates.go

```diff
@@ -234,6 +234,20 @@ func ServiceUsageServiceBeingActivated(err error) (bool, string) {
 	return false, ""
 }
 
+// See https://github.com/hashicorp/terraform-provider-google/issues/14691 for
+// details on the error message this handles
+// This is a post-operation error so it uses tpgresource.CommonOpError instead of googleapi.Error
+func ServiceUsageInternalError160009(err error) (bool, string) {
+	// a cyclical dependency between transport/tpgresource blocks using tpgresource.CommonOpError
+	// so just work off the error string. Ideally, we'd use that type instead.
+	s := err.Error()
+	if strings.Contains(s, "encountered internal error") && strings.Contains(s, "160009") && strings.Contains(s, "with failed services") {
```

Comment:
Do we need all the matching on the error string? Is this code common enough that we don't want to retry other 160009s?

I worry about the error string changing in the future

Reply:
Frankly, I'm not sure- this error probably should not have made it out to public errors in this form, so it's pretty likely we'll never see it again after a few weeks. I wanted to avoid just `160009` in case a user project name included the string or something.